labrad/test/test_units.py

```diff
@@ -194,5 +194,11 @@ def testInUnitsOf(self):
         self.assertTrue((1*s).inUnitsOf(ms) == 1000*ms)
         self.assertTrue((1*s).inUnitsOf('ms') == 1000*ms)
 
+    def testBaseUnit(self):
+        x = Value(1, 'ns^2')
+
+        self.assertTrue(x.unit.base_unit == units.Unit('s^2'))
+        self.assertTrue(x.inBaseUnits() == Value(1e-18, 's^2'))
```

Comment:
We should add a test that `Unit('ns')**2 == Unit('ns^2')`, which is where the confusion came in.